### DISCUSSION TITLE
Training a model for interactive segmentation

### DISCUSSION THREAD

**Discussion by DeboshreeR** [Project-MONAI/MONAILabel#1066] [General]
I would like to train models (described in monailabel) with our own custom dataset for interactive segmentation of tumors. Examples of training models such as Deep Net are described (https://github.com/Project-MONAI/MONAIBootcamp2021/blob/main/day1/3.%20End-To-End%20Workflow%20with%20MONAI.ipynb)

I am completely clueless on how to train a model for interactive segmentation.

**Comment by diazandr3s** [comment]
Hi @DeboshreeR,

Thanks for posting this question.

Can you please provide more details about the custom dataset? Is it an MR dataset? Do you have labelled volumes available? If not, are you familiar with 3D Slicer?

You may find useful this video tutorial: https://www.youtube.com/watch?v=8y1OBQs2wis&list=PLtoSVSQ2XzyD4lc-lAacFBzOdv5Ou-9IA
There you can see how to install MONAI Label, how to train a MONAI Label model from scratch and details about the 3D Slicer module for MONAI Label.

Hope this helps

**Comment by DeboshreeR** [comment]
Thanks for your reply. I have a set of labelled dataset (CT images) that I want to use to train the model for tumor interactive segmentation. 
I studied 3D Slicer and worked initially on monailabel dataset. I am looking to use a model pre-trained on my custom dataset for further segmentation, evaluation etc

**Reply by diazandr3s** [reply]
> Thanks for your reply. I have a set of labelled dataset (CT images) that I want to use to train the model for tumor interactive segmentation. I studied 3D Slicer and worked initially on monailabel dataset. I am looking to use a model pre-trained on my custom dataset for further segmentation, evaluation etc

Thanks for the details, @DeboshreeR.

In this [discussion](https://github.com/Project-MONAI/MONAILabel/discussions/1055#discussioncomment-3830237), you can find how you can organize the images and labels and then start the MONAI Label server.

Please don't hesitate to comment or ask any questions regarding this.

Hope this helps,

**Comment by DeboshreeR** [comment]
Thank you for your reply. I was able to train and generate a model .pth file. 
I now require to load the .pth model in Slicer for interactive segmentation. I am looking for a method to do so
